### PR TITLE
NXP-30370: add readme

### PR DESCRIPTION
**Pull request by andreacornaglia**

### CODE REVIEW

**Review by doulba — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Since it already contemplates the possibility of integrating packages/addons, it is possible to serve it with `NUXEO_PACKAGES` environment variable pointing to the desired packages/addons.
+
+
+## CI/CD
+
+Continuous Integration & Continuous Deployment(and Delivery) are an important part of the development process.
+
+Nuxeo Cold Storage integrates [Jenkins pipelines](https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-coldstorage/) for each maintenance branch, for _master_ (fast track) and also for each opened PR. 
+
```
Comment (RSalem07) [inline README.md:120]: can you replace `master` by LTS (long time suppport)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+npm run format
+```
+
+Both `lint` and `format` commands run automatically before performing a commit in order to help us keeping the code base consistent with the rules defined.
+
+#### Integration with Web UI
+
+Despite being an "independent" project, this frontend contribution is build and aims to run as part of Nuxeo Web UI. So, most of the development will be done under that context.
+To have the best experience possible, it is recommended to follow the `Web UI Development workflow` on [repository's README](https://github.com/nuxeo/nuxeo-web-ui/blob/master/README.md).
+
```
Comment (RSalem07) [inline README.md:111]: for web-ui we should point to the lts-2020 of web-ui, I think the maintenance branch is https://github.com/nuxeo/nuxeo-web-ui/tree/maintenance-3.0.x

File: README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+## CI/CD
+
+Continuous Integration & Continuous Deployment(and Delivery) are an important part of the development process.
+
+Nuxeo Cold Storage integrates [Jenkins pipelines](https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-coldstorage/) for each maintenance branch, for _master_ (fast track) and also for each opened PR. 
+
+The following features are available:
+- Possibility of having a dedicated preview environment for a PR by using the tag GitHub PR tag `preview`
+- Each PR merge to _master_ branch will generate a "release candidate" package
+- A preview aligned on _master_ branch code is available
```
Comment (RSalem07) [inline README.md:124]: could replace master by `lts-2020`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+[Web Test Runner](https://modern-web.dev/docs/test-runner/overview/) is the test runner used to run this contribution unit tests.
+The tests run against bundled versions of Chromium, Firefox and Webkit, using [Playwright](https://www.npmjs.com/package/playwright)
+
+#### Functional Tests
+
+```shell script
+npm run ftest
+```
+
+To run the functional tests, [Nuxeo Web UI Functional Testing Framework](https://github.com/nuxeo/nuxeo-web-ui/tree/master/packages/nuxeo-web-ui-ftest) is used.
```
Comment (RSalem07) [inline README.md:76]: should point to the web-ui maintenance branch https://github.com/nuxeo/nuxeo-web-ui/tree/maintenance-3.0.x/packages/nuxeo-web-ui-ftest

**Review by RSalem07 — COMMENTED** [review]
left some comments

File: README.md (hunk 3 above)
Comment (andreacornaglia) [reply README.md:124]: btw it's 2021 ;)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+## CI/CD
+
+Continuous Integration & Continuous Deployment(and Delivery) are an important part of the development process.
+
+Nuxeo Cold Storage integrates [Jenkins pipelines](https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-coldstorage/) for each maintenance branch, for _LTS_ (fast track) and also for each opened PR. 
+
+The following features are available:
+- Possibility of having a dedicated preview environment for a PR by using the tag GitHub PR tag `preview`
+- Each PR merge to _master_ branch will generate a "release candidate" package
```
Comment (RSalem07) [inline README.md:123]: _master_ to  _lts-2021_

**Review by RSalem07 — COMMENTED** [review]
remaining one last  _master_

**Review by RSalem07 — DISMISSED** [review]

**Review by RSalem07 — APPROVED** [review]

**Review by doulba — APPROVED** [review]

**Pull request merged**